packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -12,12 +12,14 @@ import {
   useOmSupplyApi,
   useNotification,
   generateUUID,
+  useTranslation,
```

Comment:
here i've updated the columns to use the created date time field rather than entry date time and translated it. I've left the header as `Entered`